### ISSUE TITLE
SC 412 -- Link text is set as a paragraph

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#23]
## URL
https://demo.fileyourstatetaxes.org/en/questions/sms-terms

## Issue
The email address is set as static text when it should be a `link`. 

## WCAG Success Criterion
[SC 4.1.2: Name, Role, Value (Level A)](https://w3.org/TR/WCAG21#name-role-value)

## User Impact
Text that is meant to be interactive must have a programmatic markup to make it interactive. A user with a physical disability for example may have difficulty transposing this information or copy and pasting into their email client. The hyperlink comes with interactivity built in. 

### Current:
```html
<li>As always, message and data rates may apply for any messages sent to you from us and to us from you. If you have any questions about your text plan or data plan, it is best to contact your wireless provider. 

For all questions about the services provided by this short code, you can send an email to <strong>help@fileyourstatetaxes.org</strong>.
</li>
``` 

### Should be:

```html
<li>As always, message and data rates may apply for any messages sent to you from us and to us from you. If you have any questions about your text plan or data plan, it is best to contact your wireless provider. 

For all questions about the services provided by this short code, you can send an email to <strong><a href="mailto:help@fileyourstatetaxes.org>help@fileyourstatetaxes.org</strong>.
</li>
```

**Comment by damiansian** [comment]
Looks good. 

<img width="1332" alt="Anchor tag used for email." src="https://github.com/user-attachments/assets/90140650-f00f-4bf7-9a45-3281766f6cd5" />
